draft-ietf-webtrans-http2.md

```diff
@@ -284,19 +284,49 @@ WebTransport streams and datagrams that belong to different WebTransport
 sessions are identified by the CONNECT stream on which they are transmitted,
 with one WebTransport session consuming one CONNECT stream.
 
-## Transport Considerations
+## Transport Properties
+
+The WebTransport framework [OVERVIEW] defines a set of optional transport
+properties that clients can use to determine the presence of features which
+might allow additional optimizations beyond the common set of properties
+available via all WebTransport protocols.
 
 Because WebTransport over TCP-based HTTP semantics relies on the underlying
 protocols to provide in order and reliable delivery, there are some notable
-differences from the way in which QUIC handles application data.
-
-Endpoints MUST send stream data in order. As there is no ordering mechanism
+differences from the way in which QUIC handles application data. For example,
+endpoints send stream data in order. As there is no ordering mechanism
 available for the receiver to reassemble incoming data, receivers assume that
 all data arriving in WT_STREAM capsules is contiguous and in order.
 
-DATAGRAM capsules are delivered to the remote WebTransport endpoint reliably,
-however this does not require that the receiving implementation deliver that
-data to the application in a reliable manner.
+Below are details about support in WebTransport over HTTP/2 for the properties
+defined by the WebTransport framework.
+
+Stream Independence:
+
+: WebTransport over HTTP/2 does not support stream independence, as HTTP/2
+  inherently has head-of-line blocking.
+
+Partial Reliability:
+
+: WebTransport over HTTP/2 does not support partial reliability, as HTTP/2
+  retransmits any lost data. This means that any datagrams sent via
+  WebTransport over HTTP/2 will be retransmitted regardless of the preference
+  of the application. The receiver is permitted to drop them, however, if it is
+  unable to buffer them.
+
+Pooling Support:
+
+: WebTransport over HTTP/2 supports pooling, as multiple transports using
+  WebTransport over HTTP/2 may share the same underlying HTTP/2 connection and
+  therefore share a congestion controller and other transport context.
```

Comment:
Is it worth pointing out that streams are not created out of a connection-wide shared pool here?  This is one area where the QUIC version is arguably worse.

Reply:
Seems reasonable, added a note in [cb29154](https://github.com/ietf-wg-webtrans/draft-webtransport-http2/pull/68/commits/cb29154ced1c98b49ac5c481ab79513e23304e2b)